src/components/modal/modal.e2e.ts

```diff
@@ -64,9 +64,9 @@ describe("calcite-modal properties", () => {
 });
 
 describe("opening and closing behavior", () => {
-  it.skip("opens and closes", async () => {
+  it("opens and closes", async () => {
     const page = await newE2EPage();
-    await page.setContent(`<calcite-modal></calcite-modal>`);
+    await page.setContent(`<calcite-modal style="transition: opacity 0.3ms"></calcite-modal>`);
```

Comment:
Cool! can you use the css variable we have for setting the transition instead of setting the style directly?